src/Printer.php

```diff
@@ -37,7 +37,7 @@ public function printData(OutputInterface $output)
         $output->writeln(PHP_EOL . $separator . PHP_EOL);
 
         $total = 0;
-        foreach ($this->fileReports as $fileReport) {
+        foreach($this->fileReports as $fileReport) {
```

Comment:
It's on purpose there to trigger a Travis CI build failing due to a coding standard violation.